frontend/src/App.js

```diff
@@ -54,6 +56,21 @@ function App() {
   const [extraPixelsUsed, setExtraPixelsUsed] = useState(0);
   const [extraPixelsData, setExtraPixelsData] = useState([]);
 
+  useEffect(() => {
+    let getExtraPixelsEndpoint = `${backendUrl}/getExtraPixels`;
```

Comment:
Change this endpoint to take `address` as a query parameter, like this `${backendUrl}/getExtraPixels?address={address}`

You can set {address} as a 0 constant for now.